sky/packages/sky/lib/src/widgets/gesture_detector.dart

```diff
@@ -27,7 +28,10 @@ class GestureDetector extends StatefulComponent {
     this.onHorizontalDragEnd,
     this.onPanStart,
     this.onPanUpdate,
-    this.onPanEnd
+    this.onPanEnd,
+    this.onScaleStart,
+    this.onScaleUpdate,
+    this.onScaleEnd
   }) : super(key: key);
```

Comment:
Scale is a superset of Pan, which means that if we try to detect them both, Scale should always win.  If that's the case, we shouldn't try to recognize them both...